src/Symfony/Component/PropertyAccess/Annotation/Property.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor configuration annotation.
+ *
+ * @Annotation
+ * @Target({"PROPERTY"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class Property
```

Comment:
There are also some advantages on using the property annotations:
- the annotation is next to the property declaration, which is very convenient when looking for it. On the other hand, method annotations are spread out in the class definition (properties are usually placed on the top of the class and method implementations at the end)
- you can override several methods with just one annotation

Property annotations are implemented with just 19 lines of code (not taking into account the empty annotation class). In fact, all new annotations are loaded from the same method `AnnotationLoader->loadClassMetadata()`, so the code will be easy to maintain and optimize as changes would be centralized.

About being harder to understand for newcomers, we could just give them some initial guidelines and let them use whatever they want, just like the YAML vs XML vs Annotation debate.

Anyway, if you feel that only method annotations must be kept, I'll do that, of course!